addons/account/models/account_bank_statement_line.py

```diff
@@ -310,6 +310,8 @@ def _check_amounts_currencies(self):
             if not st_line.amount_currency and st_line.foreign_currency_id:
                 raise ValidationError(_("You can't provide a foreign currency without specifying an amount in "
                                         "'Amount in Currency' field."))
+            if not st_line.amount:
+                raise ValidationError(_("The value of 'Amount' cannot be Zero"))
```

Comment:
You can't do that. It's quite common to receive empty transaction from your bank containing "Happy New Year" in january ;)